app/src/editor/canvas/components/ModuleSearch.jsx

```diff
@@ -114,14 +152,29 @@ export class ModuleSearch extends React.PureComponent<Props, State> {
 
     componentDidMount() {
         window.addEventListener('keydown', this.onKeyDown)
+        this.addOrRemoveDropListener(true)
         this.load()
     }
 
     componentWillUnmount() {
         this.unmounted = true
+        this.addOrRemoveDropListener(false)
         window.removeEventListener('keydown', this.onKeyDown)
     }
 
+    addOrRemoveDropListener= (add: boolean) => {
+        const canvasElement = document.querySelector(`.${CanvasStyles.Modules}`)
+        if (canvasElement) {
+            if (add) {
+                canvasElement.addEventListener('dragover', (e: DragEvent) => { e.preventDefault() })
+                canvasElement.addEventListener('drop', (e: DragEvent) => { onDrop(e, this.addModule) })
+            } else {
+                canvasElement.removeEventListener('dragover', (e: DragEvent) => { e.preventDefault() })
+                canvasElement.removeEventListener('drop', (e: DragEvent) => { onDrop(e, this.addModule) })
```

Comment:
This `removeEventListener` can never work because both `addEventListener` & `removeEventListener` are being passed a freshly created arrow function.